staging/src/k8s.io/api/admissionregistration/v1/types.go

```diff
@@ -563,3 +579,25 @@ type ServiceReference struct {
 	// +optional
 	Port *int32 `json:"port,omitempty" protobuf:"varint,4,opt,name=port"`
 }
+
+// MatchCondition represents a condition which must by fulfilled for a request to be sent to a webhook.
+type MatchCondition struct {
+	// Name is an identifier for this match condition, used for strategic merging of MatchConditions,
+	// as well as providing an identifier for logging purposes. A good name should be descriptive of
+	// the associated expression.
+	// Name must be a valid RFC 1123 DNS subdomain, and unique in a set of MatchConditions.
+	// Required.
+	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
```

Comment:
Sorry I missed this one while kep review. For extra info, we considered to have `name` in ValidatingAdmissionPolicy and dropped it while API review. VAP has a message field which is for returning meaningful info back to use, and the expressions are expected to be unique. I am not sure if this name is used anywhere? If it is only for logging/metrics purpose, the current webhook generate name for this purpose if I remember correctly. And maybe we could consider doing the same? cc @tallclair